### PR TITLE
Add documentation for property bag entity types.

### PR DESCRIPTION
**Pull request by AndriySvyryd**
Part of #2720

### CODE REVIEW

File: entity-framework/core/modeling/shadow-properties.md [hunk 1]
```diff
@@ -43,3 +43,12 @@ var blogs = context.Blogs
 ```
 
 Shadow properties cannot be accessed after a no-tracking query since the entities returned are not tracked by the change tracker.
+
+## Property bag entity types
+
+> [!NOTE]
+> Support for Property bag entity types was added in EF Core 5.0.
+
+Entity types that contain only indexer properties are known as property bag entity types. These entity types don't have shadow properties. Currently only `Dictionary<string, object>` is supported as a property bag entity type. This means that it must be configured as a shared entity type with a unique name and the corresponding `DbSet` property must be implemented using a `Set` call.
```
Comment (smitpatel) [inline entity-framework/core/modeling/shadow-properties.md:52]: Where do we document indexer properties?

Comment (AndriySvyryd) [reply entity-framework/core/modeling/shadow-properties.md:52]: Same page

Comment (AndriySvyryd) [reply entity-framework/core/modeling/shadow-properties.md:52]: https://github.com/dotnet/EntityFramework.Docs/issues/2018

Comment (smitpatel) [reply entity-framework/core/modeling/shadow-properties.md:52]: So you updated title without adding docs for indexer properties. I got confused in that.

**Review by smitpatel — APPROVED** [review]

**Pull request merged**